docs/content/en/install/local/_index.md

```diff
@@ -1,441 +1,2316 @@
+          Type:                    ReplacingNodes
+          Last Transition Time:    2022-01-31T19:12:33Z
+          Message:
+          Reason:
+          Status:                  False
+          Type:                    Updating
+          Last Transition Time:    2022-01-31T19:12:33Z
+          Message:
+          Reason:
+          Status:                  False
+          Type:                    RollingRestart
+          Last Transition Time:    2022-01-31T19:12:33Z
+          Message:
+          Reason:
+          Status:                  False
+          Type:                    Resuming
+          Last Transition Time:    2022-01-31T19:12:33Z
+          Message:
+          Reason:
+          Status:                  False
+          Type:                    ScalingDown
+          Last Transition Time:    2022-01-31T19:12:33Z
+          Message:
+          Reason:
+          Status:                  True
+          Type:                    Valid
+          Last Transition Time:    2022-01-31T19:12:33Z
+          Message:
+          Reason:
+          Status:                  True
+          Type:                    Initialized
+          Last Transition Time:    2022-01-31T19:12:33Z
+          Message:
+          Reason:
+          Status:                  True
+          Type:                    Ready
+        Last Server Node Started:  2022-01-31T19:11:12Z
+        Node Statuses:
+          demo-dc3-default-sts-0:
+            Host ID:  2cceff49-6df2-4045-8e04-6ce262bd6fc4
+          demo-dc3-default-sts-1:
+            Host ID:  018bfd17-8a77-43c6-859b-ab69c1fc8a66
+          demo-dc3-default-sts-2:
+            Host ID:          38438f65-b10a-4b3f-a56f-488536bf4cd3
+        Observed Generation:  1
+        Quiet Period:         2022-01-31T19:12:39Z
+        Super User Upserted:  2022-01-31T19:12:34Z
+        Users Upserted:       2022-01-31T19:12:34Z
+Events:                       <none>
+```
+
+### Extract credentials
+
+On the control plane, use the following commands to extract the username and password.
+
+```bash
+kubectx kind-k8ssandra-0
+
+CASS_USERNAME=$(kubectl get secret demo-superuser -n k8ssandra-operator -o=jsonpath='{.data.username}' | base64 --decode)
+
+echo $CASS_USERNAME
+```
+
+**Output:**
+```bash
+demo-superuser
+```
+
+Now obtain the password secret:
+
+```bash
+CASS_PASSWORD=$(kubectl get secret demo-superuser -n k8ssandra-operator -o=jsonpath='{.data.password}' | base64 --decode)
+
+echo $CASS_PASSWORD
+```
+
+**Output example - your value will be different:**
+```bash
+KT-ROFfbD-O9BzWS3Lxq
+```
+
+{{% alert title="Tip" color="success" %}}
+You'll use the extract credentials for subsequent authentication in deployed containers.
+{{% /alert %}}
+
+### Verify cluster status
+
+On one of the data plane clusters, verify the cluster status. Example:
+
+```bash
+kubectx kind-k8ssandra-1
+
+$ kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -- /bin/bash
```

Comment:
```suggestion
kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- nodetool -u demo-superuser -pw 3CAvGWc4mRna8tODJgeN status
```

Just had the wrong command here, switching over to this will get the nodetool status command run automatically.

I also added the `-c cassandra` to force the container -- I realize that part was omitted from own previous docs, it's not totally required since it'll be the first container, but this will force it and make sure you don't have to updating this later if that should ever change in terms of default ordering.

Reply:
Thanks - inc'd in next commit.